protocol/cosmic-features.json

```diff
@@ -0,0 +1,927 @@
+      "0032-PRIM-033",
+      "0032-PRIM-034",
+      "0032-PRIM-035",
+      "0032-PRIM-036",
+      "0032-PRIM-037",
+      "0032-PRIM-038",
+      "0033-OCAN-011",
+      "0033-OCAN-012",
+      "0033-OCAN-013",
+      "0033-OCAN-014",
+      "0033-OCAN-015",
+      "0033-OCAN-016",
+      "0033-OCAN-017",
+      "0034-PROB-003",
+      "0034-PROB-008",
+      "0034-PROB-010",
+      "0039-MKTD-020",
+      "0039-MKTD-021",
+      "0039-MKTD-022",
+      "0039-MKTD-023",
+      "0039-MKTD-024",
+      "0039-MKTD-025",
+      "0039-MKTD-026",
+      "0039-MKTD-027",
+      "0039-MKTD-028",
+      "0039-MKTD-029",
+      "0039-MKTD-030",
+      "0039-MKTD-031",
+      "0039-MKTD-032",
+      "0039-MKTD-033",
+      "0043-MKTL-005",
+      "0043-MKTL-006",
+      "0043-MKTL-007",
+      "0043-MKTL-008",
+      "0049-TVAL-007",
+      "0049-TVAL-008",
+      "0049-TVAL-009",
+      "0049-TVAL-010",
+      "0049-TVAL-011",
+      "0049-TVAL-012",
+      "0051-PROD-004",
+      "0051-PROD-005",
+      "0051-PROD-006",
+      "0052-FPOS-003",
+      "0052-FPOS-004",
+      "0054-NETP-007",
+      "0054-NETP-008",
+      "0054-NETP-009",
+      "0054-NETP-010",
+      "0054-NETP-011",
+      "0057-TRAN-063",
+      "0065-FTCO-005",
+      "0065-FTCO-006",
+      "0065-FTCO-007",
+      "0065-FTCO-008",
+      "0066-VALW-004",
+      "0066-VALW-005",
+      "0068-MATC-060",
+      "0068-MATC-061",
+      "0068-MATC-062",
+      "0068-MATC-063",
+      "0068-MATC-064",
+      "0068-MATC-065",
+      "0068-MATC-066",
+      "0068-MATC-067",
+      "0068-MATC-068",
+      "0068-MATC-069",
+      "0068-MATC-070",
+      "0068-MATC-071",
+      "0068-MATC-072",
+      "0068-MATC-073",
+      "0068-MATC-074",
+      "0068-MATC-075",
+      "0068-MATC-076",
+      "0068-MATC-077",
+      "0068-MATC-078",
+      "0068-MATC-079",
+      "0068-MATC-080",
+      "0068-MATC-081",
+      "0068-MATC-082",
+      "0068-MATC-083",
+      "0068-MATC-084",
+      "0068-MATC-085",
+      "0068-MATC-086",
+      "0068-MATC-087",
+      "0068-MATC-088",
+      "0068-MATC-089",
+      "0068-MATC-090",
+      "0068-MATC-091",
+      "0068-MATC-092",
+      "0070-MKTD-008",
+      "0070-MKTD-009",
+      "0070-MKTD-010",
+      "0070-MKTD-011",
+      "0070-MKTD-012",
+      "0070-MKTD-013",
+      "0070-MKTD-014",
+      "0070-MKTD-015",
+      "0073-LIMN-087",
+      "0073-LIMN-090",
+      "0073-LIMN-091",
+      "0073-LIMN-092",
+      "0073-LIMN-093",
+      "0073-LIMN-094",
+      "0073-LIMN-095",
+      "0073-LIMN-096",
+      "0073-LIMN-097",
+      "0073-LIMN-098",
+      "0073-LIMN-099",
+      "0073-LIMN-100",
+      "0073-LIMN-101",
+      "0073-LIMN-102",
+      "0073-LIMN-103",
+      "0073-LIMN-104",
+      "0074-BTCH-011",
+      "0074-BTCH-012",
+      "0074-BTCH-015",
+      "0074-BTCH-016",
+      "0074-BTCH-019",
+      "0079-TGAP-006",
+      "0079-TGAP-007",
+      "0081-SUCM-033",
+      "0081-SUCM-034"
+    ]
+  },
+  "Fixed Size Amendments": {
```

Comment:
Not a cosmic feature, should probably be removed.

Reply:
Happy to remove, I just archived the old features.json as it was so we have a record of how it looked.